physionet-django/notification/templates/notification/email/mailto_notify_credential_request.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}{% autoescape off %}
+Dear {{ applicant_name }},
+
+Thank you for submitting your data use agreement. If it is approved, you will be authorized to use restricted-access PhysioNet clinical databases.
+
+Below is a copy of the agreement you submitted. Please review it for accuracy, looking especially for inaccuracies caused by browser auto-fill.
+
+It may take a week or longer to process your request. Thank you for your understanding and patience.
+
+----------------
```

Comment:
This should be imported from the DUA that they sign I guess, but again we can deal with this later!

Reply:
Yes it is. I forgot to remove this file.